db/migrate/20180517164500_create_disability_compensation_submissions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDisabilityCompensationSubmissions < ActiveRecord::Migration
+  def change
+    create_table :disability_compensation_submissions do |t|
+      t.uuid :user_uuid, null: false
+      t.string :form_type, null: false
+      t.string :claim_id, null: false, unique: true
```

Comment:
@egarreau looking at the evss swagger I think I should make this an integer, but in the [VCR cassette](https://github.com/department-of-veterans-affairs/vets-api/blob/1efa304306c9534425b2fa63bd65f7e907aa22e6/spec/support/vcr_cassettes/evss/disability_compensation_form/submit_form.yml#L70) the value is `-6820564985530150012` which would be rejected as a postgres int. Did you ever ask EVSS about this?

Reply:
Ah, I have not asked EVSS about that; will do.

Reply:
Per Tim Barto: the negative number is specific to the CI (development) instance of the API; in prod it will be positive. He's working on fixing it in CI to keep it consistent.